application/src/test/java/com/sanction/lightning/resources/FacebookResourceTest.java

```diff
@@ -172,14 +172,28 @@ public void testGetMediaBytes() {
   /* Publish Tests */
   @Test
   public void testPublishWithNullUsername() {
-    Response response = resource.publish(key, null, null, null, null);
+    Response response = resource.publish(key, null, null, null, null, null, null);
+
+    assertEquals(response.getStatusInfo(), Response.Status.BAD_REQUEST);
+  }
+
+  @Test
+  public void testPublishWithNullType() {
+    Response response = resource.publish(key, null, null, null, null, null, null);
+
+    assertEquals(response.getStatusInfo(), Response.Status.BAD_REQUEST);
```

Comment:
This test is exactly the same as `testPublishWithNullUsername`. I think you meant to pass in a non-null username into `resource.publish()`